modules/consensus/synchronize.go

```diff
@@ -380,6 +405,11 @@ func (cs *ConsensusSet) rpcRelayHeader(conn modules.PeerConn) error {
 	if err == errOrphan {
 		// If the header is an orphan, try to find the parents.
 		go func() {
```

Comment:
Is there a reason that we want the handling of `errOrphan` to be done in a separate goroutine?